research/RSOC.ipynb

```diff
@@ -0,0 +1,755 @@
+{
```

Comment:
<p>Śmiesznie to wygląda! Ładny syntax.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SocSIM/SocSIM/pull/63/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>